e2e/support/cypress_data.js

```diff
@@ -162,6 +162,34 @@ export const USERS = {
       { id: COLLECTION_GROUP, is_group_manager: false },
     ],
   },
+  dbRouterUserA: {
```

Comment:
I would hesitate to add users for a specific test suite to the general test setup